### DISCUSSION TITLE
umi v2.0 存在colors.js这类的问题吗

### DISCUSSION THREAD

**Discussion by ReinhardChen93** [umijs/umi#7775] [Q&A]
今天检查打包依赖发现存在colors.js,是否是build时候加入的,会影响打包后的项目吗

**Comment by leftstick** [comment]
我感觉打包应该不会有影响，在umi这类"编译时"框架里，colors.js 经常是用在 terminal 输出内容的美化上，和『编译、打包源码』关系不大。
当然，我没验证过，仅仅是推断

**Comment by xiaohuoni** [comment]
我记忆中应该是没用到 colors.js 才对，建议升级到 3 或者 4(再等几天就发布了)，2版本没有锁包策略，没有 colors.js 也可能会有其他类似的问题。
